src/app/views/templates/FormDeactivatedNotification.tsx

```diff
@@ -0,0 +1,25 @@
+import { Body, Head, Html, Text, Link } from '@react-email/components'
+import { FormDeactivatedNotificationHtmlData } from '../../services/mail/mail.types'
+
+export const FormDeactivatedNotification = ({
+  formTitle,
+  formLink,
+  appName,
+}: FormDeactivatedNotificationHtmlData): JSX.Element => {
+  return (
+    <Html>
+      <Head />
+      <Body>
+        <Text>Dear form admins(s),</Text>
+        <Text>
+            Your <b>{formTitle}</b> {' '} (<a href={formLink}>{formLink}</a>)
+            has been closed to new responses because it has exceeded the number of free SMSes.
+            Please fill out this form (<a href="https://go.gov.sg/formsg-support">go.gov.sg/formsg-support</a>)
```

Comment:
Can reuse constants!
```suggestion
import { SUPPORT_FORM_LINK } from 'shared/constants'
...
Please fill out this form (<a href={SUPPORT_FORM_LINK}>go.gov.sg/formsg-support</a>)
```

Reply:
Missed this constant - I've replaced this as well as the other area(s) that link to the form support